src/cljs/ui/subscriptions.cljs

```diff
@@ -93,11 +93,6 @@
       (merge ongoing-analysis
              (get-in db [:analysis id])))))
 
-(re-frame/reg-sub
```

Comment:
I already commented on this, but is there a good reason to change to ratom?

Reply:
answered before